fdbserver/KeyValueStoreMemory.actor.cpp

```diff
@@ -462,39 +461,55 @@ class KeyValueStoreMemory final : public IKeyValueStore, NonCopyable {
 		return total;
 	}
 
-	// Data format for normal operation:
+	static bool isOpEncrypted(OpHeader* header) { return header->op >> 31 == 1; }
+
+	static void setEncryptFlag(OpHeader* header, bool set) {
+		if (set) {
+			header->op |= (1UL << 31);
```

Comment:
nit: make `31` and #define , for some reason we update the bit-field; it would be easy to update all its usage (without silent error introductions). 
Thanks